.github/workflows/build-deploy-pudl.yml

```diff
@@ -45,7 +55,7 @@ jobs:
 
       - name: Docker Metadata
         id: docker_metadata
-        uses: docker/metadata-action@v5.3.0
+        uses: docker/metadata-action@v5
```

Comment:
I removed the minor versions to reduce the frequency with which dependabot makes PRs. We haven't (yet) had issues with github actions breaking during minor version updates.